### PR TITLE
feat: added navigation component

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added 2 navigation bars:

>Simple Nav Bar
Nav Bar with shadow

naviga

### CODE REVIEW

**Review by AnuragSahu11 — APPROVED** [review]
keep up the good work, nice choice of colors

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.nav-bar{
+    align-items: center;
+    background-color: var(--white);
+    border-radius: 5px;
+    display: flex;
+    justify-content: space-between;
+    padding:var(--size-12);
+    height: 5rem;
+
```
Comment (saadahmadhere) [inline components/navigation.css]: remove extra space here

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    
+}
+.nav-section-left{
+    align-items: center;
+    display: flex;
+    gap:10px;
+    padding-left:var(--size-20);
+}
+
+
```
Comment (saadahmadhere) [inline components/navigation.css:18]: and here

File: components/navigation.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+.nav-section-right{
+    align-items: center;
+    display: flex;
+    gap:var(--size-16);
+    padding-right:var(--size-12);
+}
```
Comment (saadahmadhere) [inline components/navigation.css:24]: give a space between 2 classes

File: components/navigation.css (hunk 1 above)
Comment (saadahmadhere) [inline components/navigation.css]: would look good if you remove extra space here

File: components/navigation.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+       width: 100vw;
+       padding: 0;
+   }
+   .nav-section-left{
+       gap:0;
+   }
+   .nav-section-right{
+       gap:7px;
+   }
+
```
Comment (saadahmadhere) [inline components/navigation.css]: remove space from here

**Review by saadahmadhere — COMMENTED** [review]
Just take care of the spacing. Rest of the code looks pretty.

File: components/navigation.css (hunk 1 above)
Comment (GuptaEesh) [reply components/navigation.css]: done

File: components/navigation.css (hunk 3 above)
Comment (GuptaEesh) [reply components/navigation.css:24]: done

File: components/navigation.css (hunk 4 above)
Comment (GuptaEesh) [reply components/navigation.css]: done

File: components/navigation.css (hunk 2 above)
Comment (GuptaEesh) [reply components/navigation.css:18]: done

File: components/navigation.css (hunk 1 above)
Comment (GuptaEesh) [reply components/navigation.css]: done

**Comment by GuptaEesh** [comment]
Used prettier for spacing. Thanks for the suggestion. Got 2 reviews. Merging now.

**Pull request merged**